src/ansys/mapdl/core/_commands/post26/set_up.py

```diff
@@ -0,0 +1,1668 @@
+             - x, y, or z relative angular velocity.
+           * - ACC
+             - X, Y, Z
+             - x, y, or z relative linear acceleration.
+           * - DMG
+             - X, Y, Z
+             - x, y, or z relative angular acceleration.
+           * - RF
+             - X, Y, Z
+             - Relative reaction forces in the local x, y, or z direction.
+           * - RM
+             - X, Y, Z
+             - Relative reaction moments in the local x, y, or z direction.
+
+        """
+        command = f"JSOL,{nvar},{elem},{item},{comp},{name}"
+        return self.run(command, **kwargs)
+
+    def store(
+        self,
+        lab: str = "",
+        npts: str = "",
+        freq: str = "",
+        toler: str = "",
+        cluster: str = "",
+        **kwargs,
+    ):
+        r"""Stores data in the database for the defined variables.
+
+        Mechanical APDL Command: `STORE <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_STORE.html>`_
+
+        Parameters
+        ----------
+        lab : str
+            Valid labels:
+
+            * ``MERGE`` - Merge data from results file for the time points in memory with the existing data
+              using current specifications (default).
+
+            * ``NEW`` - Store a new set of data, replacing any previously stored data with current result file
+              specifications and deleting any previously-calculated variables (see ).
+
+              Variables defined using the :ref:`ansol` command are also deleted if they represent element-based
+              results. Variables created using `nodal-averaged results
+              <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_elem/Hlp_E_CH2_2.html#nodeavepost>`_ are
+              not deleted.
+
+            * ``APPEN`` - Append data from results file to the existing data.
+
+            * ``ALLOC`` - Allocate (and zero) space for ``NPTS`` data points.
+
+            * ``PSD`` - Create a new set of frequency points for PSD calculations (replacing any previously
+              stored data and erasing any previously calculated data).
+
+        npts : str
+            The number of time points (or frequency points) for storage (used only with ``Lab`` = ALLOC or
+            PSD). The value may be input when using POST26 with data supplied from other than a results
+            file. This value is automatically determined from the results file data with the NEW, APPEN, and
+            MERGE options. For the PSD option, ``NPTS`` determines the resolution of the frequency vector
+            (valid numbers are between 1 and 10, defaults to 5).
+
+        freq : str
+            A frequency value, or an array containing frequency values (Hz). Use :ref:`dim` to define the
+            array and enclose the array name in percent signs (for example, :ref:`store`,,,,``%arrayname``
+            %). A default value of 1% of damping is considered for clustering around the user-input
```

Comment:
Probably a typo in the original docs but:

<img width="331" alt="image" src="https://github.com/user-attachments/assets/aa0a6d66-56b8-4ea4-8bd3-1774fce7669f" />

Reply:
the weird character order has been fixed. But the format issue remains, I am not sure if we are skipping this as the italics?

<img width="478" alt="image" src="https://github.com/user-attachments/assets/4ef8fd44-77a3-4575-9700-6e280557cd56" />

Reply:
I hadn't checked the rendering. I can remove the ` %` if you prefer.